### PR TITLE
Reenable CTFont CaseInsensitive test

### PR DESCRIPTION
**Pull request by aballway**
Makes CTFontCreateWithName case insensitive

### CODE REVIEW

File: Frameworks/CoreText/DWriteWrapper.mm [hunk 1]
```diff
@@ -641,14 +641,22 @@ CFStringRef _DWriteGetFamilyNameForFontName(CFStringRef fontName) {
                           CFArrayRef fontNames = _DWriteGetFontNamesForFamilyName(familyName);
 
                           for (size_t j = 0; j < CFArrayGetCount(fontNames); j++) {
-                              CFDictionaryAddValue(initMap, static_cast<CFStringRef>(CFArrayGetValueAtIndex(fontNames, j)), familyName);
+                              CFStringRef systemFontName = static_cast<CFStringRef>(CFArrayGetValueAtIndex(fontNames, j));
+                              CFMutableStringRef upperSystemFontName =
+                                  CFStringCreateMutableCopy(nullptr, CFStringGetLength(systemFontName), systemFontName);
+                              CFAutorelease(upperSystemFontName);
+                              CFStringUppercase(upperSystemFontName, CFLocaleCopyCurrent());
```
Comment (ms-jihua) [inline Frameworks/CoreText/DWriteWrapper.mm]: >CFLocaleCopyCurrent() [](start = 69, length = 21)

locale leaks here and below

Comment (ms-jihua) [inline Frameworks/CoreText/DWriteWrapper.mm]: nit: you could've also used CFLocaleGetSystem() but whatever

---
In reply to: [80806159](https://github.com/Microsoft/WinObjC/pull/1042#discussion_r80806159) [](ancestors = 80806159)

**Comment by ms-jihua** [comment]
:shipit:

**Comment by mnithish** [comment]
:shipit:

**Comment by rajsesh-msft** [comment]
:shipit:

**Pull request merged**